include/ck/utility/data_type.hpp

```diff
@@ -965,47 +965,21 @@ inline __host__ __device__ constexpr float type_convert<float, bhalf_t>(bhalf_t
 }
 
 // convert fp32 to bfp16
+struct uint16x2_t{
```

Comment:
This definition ```uint16x2_t``` need to be inside  ```inline __host__ __device__ constexpr bhalf_t type_convert<bhalf_t, float>(float x)```, putting it in ```ck``` namespace may conflict with other definition